contrib/podmanremoteimage/README.md

```diff
@@ -0,0 +1,26 @@
+podman-remote-images
+====================
+
+Overview
+--------
+
+This directory contains the containerfile for creating a container image which consist podman-remote binary
+for each platform (win/linux/mac). Plan is to hook it with quay github build trigger to generate this image
+for each PR merged (for any branch or tag).
```

Comment:
The current build script probably won't work for mac/windows since it runs on a Fedora VM w/ nested-virt emulation (for multi-arch).  Though I could be wrong, I'm assuming we need a mac or windows VM to build for those platforms, no?

Reply:
Perhaps it's best not to mention "plans" in readme files.  Better to just file an RFE issue so the intent isn't lost.

Reply:
@cevich Even for podman-remote binaries we need windows/mac vm? I think it is cross compiled so should work on Fedora VM w/ nested-virt emulation (might be slow but should work)?

Reply:
Nope, my understanding is wrong, [see my next comment below](https://github.com/containers/podman/pull/15813#pullrequestreview-1109266061).